components/nds-tabs/src/Tabs.js

```diff
@@ -161,5 +184,9 @@ export class Tabs extends Component {
 }
 
 Tabs.propTypes = {
-	children: PropTypes.oneOfType([PropTypes.arrayOf(Tab), Tab]).isRequired
+	children: PropTypes.oneOfType([PropTypes.arrayOf(Tab), Tab]).isRequired,
+	initialTab: PropTypes.number,
+	changeCallback: PropTypes.func,
```

Comment:
Could this be more 'reacty' and consistent with other events elsewhere to be `onChange`?
```suggestion
	onChange: PropTypes.func,
```

Reply:
I wondered about this. So it's not technically an 'onChange' in the traditional sense, as in it's not an input. So I thought it might be clearer to specify that it's a callback to be explictly defined in the parent...?

Reply:
Maybe `onTabChange` makes more sense then?